<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findcontacts</class>
 <widget class="QWidget" name="findcontacts">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="avatar_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Icons/Resource/Icon/ic_profile_set.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="add_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add contact:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_lay">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>User is not found..</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineedit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
border-bottom-color: rgb(0, 0, 0);
border-style: outset;
border-top-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="invite_cont_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color: black; background: white;}
QPushButton:hover{background: #f6f4ff; border:none;}</string>
   </property>
   <property name="text">
    <string>Invite</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_register.png</normaloff>:/Icons/Resource/Icon/ic_register.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
